data_describe/core/summary.py

```diff
@@ -60,14 +60,14 @@ def show(self, viz_backend=None, **kwargs):
 
 
 def data_summary(data, compute_backend=None):
-    """Summary statistics and data description.
+    """Summary statistics and data description with metrics in rows and original fields as columns.
```

Comment:
If I were to be more picky about the docstring, the addition here feels like more of a detail than a summary of what the function does. As such, I would reformat it as:

```
"""Summary statistics and data description.

The default output has metrics in rows and the original (input) fields as columns.

Args:
    ...
"""
```

... and if the current wording isn't clear we can add more information to explain what the function does.